core/drivers/clk/sam/at91_cpu_opp.c

```diff
@@ -118,6 +118,10 @@ static TEE_Result opp_rates_setup(const void *fdt, int node)
 
 		rate_num++;
 	}
+
+	/* Ensure rates are increasing ordered */
```

Comment:
"in ascending order" is the commonly used term

Reply:
ok, thanks.